addons/mass_mailing/static/src/js/mailing_contact_kanban.js

```diff
@@ -0,0 +1,33 @@
+/** @odoo-module **/
+
+import KanbanController from 'web.KanbanController';
+import KanbanView from 'web.KanbanView';
+import viewRegistry from 'web.view_registry';
+import {_t} from 'web.core';
+
+var MailingContactController = KanbanController.extend({
+    buttons_template: 'MailingContactKanbanView.buttons',
+
+    events: _.extend({}, KanbanController.prototype.events, {
```

Comment:
Someone told me it was better to use `Object.assign()` instead of `_.extend()` :smile:

Reply:
Thanks for the tips :)